src/vos/evtree.c

```diff
@@ -2639,8 +2639,8 @@ evt_ent_array_fill(struct evt_context *tcx, enum evt_find_opc find_opc,
 			if (find_opc == EVT_FIND_OVERWRITE)
 				has_agg = true;
 
-			evt_rect_overlap(&rtmp, rect, &range_overlap,
-					 &time_overlap);
+			evt_rect_overlap(&rtmp, rect, &range_overlap, &time_overlap,
+					 find_opc == EVT_FIND_SAME);
```

Comment:
I think the rebuild minor epoch check should only be applied when find_opc == EVT_FIND_OVERWRITE? Why do we need to apply it to EVT_FIND_ALL & EVT_FIND_FIRST?